website/docs/voor-maintainers/werkafspraken.md

```diff
@@ -0,0 +1,19 @@
+# Werkafspraken
+
+In dit document staan de werkafspraken die niet passen in linters, codecheckers, build configuraties of andere automatische tooling.
+
+Het is niet de bedoeling dat werkafspraken óók als automatische check bestaan. Hiermee willen we voorkomen dat de werkafspraken uit de pas lopen met automatische checks.
+
+## Voertaal
+
+Tussen de diverse communicatiekanalen (Slack, code, GitHub) en gebruikers (Afnemers, maintainers) heerst een spanningsveld mbt. de voertaal.
+
+Uitgangspunt is: De voertaal van de DSO Toolkit is Nederlands, tenzij:
+
+- Namen van componenten zijn Engels.
+- (Lokale) code is Engels, uitgezonderd domeinspecifieke termen (zoals Renvooi).
```

Comment:
Wil je hier niet ook de comments in de code benoemen? ZIjn die ook Engels of Nederlands? We hebben momenteel 2 smaakjes in comments: Nederlandse en Engelse. Bijv. de `annotation-*.tsx` hebben NL comments, terwijl het meerendeel EN comments hebben. 🤷

Reply:
Maar wat willen we eigenlijk: JSDoc is wat mij betreft Nederlands. Gaan we commentaar voor ons (maintainers) ook Nederlands doen?

Reply:
JSDoc is momenteel in het merendeel van de webcomponenten in het Engels. Da's dan best opvallend. Willen we die dan allemaal omschrijven naar NL?  En alle overige comments ook omzetten naar NL? Ik ben wel gewend dat comments altijd in het Engels zijn.